crash_similarity.py

```diff
@@ -85,7 +105,15 @@ def get_stack_trace_for_uuid(uuid):
     return data['proto_signature']
 
 
-def train_model(corpus):
+ def train_model(corpus):
+    
+    # Store the time of training the model in last_trained.txt
+    cur_time = datetime.datetime.today() 
+    str = cur_time.strftime('%b %d %Y %I:%M%p')
```

Comment:
You shouldn't redefine `str`. `str` is builin string class
Anyway, you doesn't need additional variable containing string here